themes/poshmon.omp.json

```diff
@@ -0,0 +1,126 @@
+{
+  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
+  "blocks": [
+    {
+      "alignment": "left",
+      "segments": [
+        {
+          "background": "#2e9599",
+          "foreground": "#ffffff",
+          "leading_diamond": "\ue0b6",
+          "powerline_symbol": "\ue0b0",
+          "style": "diamond",
+          "template": "\uf91c {{ .HostName }} ",
```

Comment:
Add visual fix changing line 13 to: `"template": "\uf91c {{ .HostName }}\u2800",`

Reply:
![image](https://user-images.githubusercontent.com/40523695/178349006-fa80f628-7477-4c56-9ce8-9a640c26199d.png)

Reply:
@the-eduardo this fix is to fix a bug in Windows Terminal and that's going to provide cross platform inconsistency (as it prints characters elsewhere).